examples/window/Window.cs

```diff
@@ -1,169 +0,0 @@
-using System;
```

Comment:
No point in removing examples that work on other platforms as mentioned previously.